components/Contact.js

```diff
@@ -46,9 +46,9 @@ const Contact = () => {
               />
             </Form.Group>
           </Form.Row>
-          <HomeMadeButton type="purpleWhite" link="/">
+          {/* <HomeMadeButton type="purpleWhite" link="/">
```

Comment:
are you planning to remove the comments later or are you gonna keep it there for now?

Reply:
I think I was just testing and forgot to comment back in